core/modules/notifications/client/notifications.hbs.html

```diff
@@ -32,6 +32,9 @@
   {{#modal id="displayNotificationsPanel" title="Notifications"}}
     <div class="header">
       <h2 class="h2">Notifications</h2>
+      <h4 class="h4">
+        <button type="button" class="mark-as-read js-notification-mark-all-as-read">Mark all as read</button>
+      </h4>
```

Comment:
Maybe an icon + hover à la Gmail would be nicer?

<img width="142" alt="image" src="https://user-images.githubusercontent.com/835072/192994012-649e668f-9267-4447-8fd2-186027fc30ba.png">

Reply:
![image](https://user-images.githubusercontent.com/34689945/193026253-860d7cb8-52df-46df-87f0-f34332bc3f01.png)
Added in: https://github.com/Refty/lemverse/pull/26/commits/74b730f9294d3fca8bbfca203cbe84503c1b7229

Reply:
nice